### PR TITLE
Fix styles for AdobeStok, IMprove AdobeStock Buttons to check configuration

### LINKED ISSUE #1324: The  "...exceeds maximum file size limit." error message doesn't disappear within several seconds
### Precondition (*)
1. Installed Magento with Adobe Stock Integration

### Steps to reproduce (*)
1. From Admin go to **Content** - **Pages**, click **Add New Page**
2. Expand **Content**,click **Show / Hide Editor**, click **Insert Image...**
3. Select a folder from the folder tree 
4. Click **Upload Image** button
5. Select an image file from your local machine, which has size over 2 MB, and click Open
6. Verify if the appeared _"cannot upload 'filename"... exceeds maximum file size limit."_ message disappears within several seconds

### Expected result (*)
The error message disappears

### Actual result (*)
The error message doesn't disappear
fiel_size_limit_dissapear

### Note
- the _"unsupported file format"_ error message disappears as expected

### LINKED ISSUE #1326: The "Search Adobe Stock" button is available in Enhanced Media Gallery when the Adobe Stock Integration is disabled
### Precondition (*)
1. Installed Magento with Adobe Stock Integration
2. Disabled Adobe Stock Integration in _Stores - Configuration  - Advanced - System - Adobe Stock Integration_ 
3. Enhanced Media Gallery in Enabled 

### Steps to reproduce (*)
1. From Admin go to **Content** - **Pages**, click **Add New Page**
2. Expand **Content**,click **Show / Hide Editor**, click **Insert Image...** 
3. Verify if the **Search Adobe Stock** button is available

### Expected result (*)
The Search Adobe Stock button is not available as the integration is disabled

### Actual result (*)
The Search Adobe Stock button. There could be a situation when the Integration disabled, but user is still able to use Adobe Stock functionality, Save preview, License...

### Note
- not reproducible with not Enhanced media Gallery
- test on Cucumber Studio https://studio.cucumber.io/projects/131313/test-plan/folders/943908/scenarios/4847771

### LINKED ISSUE #1327: The Configuration link is not visible in Adobe Stock window
### Precondition (*)
1. Installed Magento with Adobe Stock Integration
2. Enabled Adobe Stock Integration in _Stores - Configuration  - Advanced - System - Adobe Stock Integration_ but the API keys are not provided
3. Enhanced Media Gallery in Enabled 

### Steps to reproduce (*)
1. From Admin go to **Content** - **Pages**, click **Add New Page**
2. Expand **Content**,click **Show / Hide Editor**, click **Insert Image...** 
3. Click **Search Adobe Stock** button 

### Expected result (*)
configuration_link_expected

### Actual result (*)
The configuration link is not visible
configuration_link

### PR DESCRIPTION
**Pull request by Nazar65**
### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#1326: The "Search Adobe Stock" button is available in Enhanced Media Gallery when the Adobe Stock Integration is disabled
2. magento/adobe-stock-integration#1327: The Configuration link is not visible in Adobe Stock window
3.  magento/adobe-stock-integration#1324:  The "...exceeds maximum file size limit." error message doesn't disappear within several seconds

### Manual testing scenarios (*)
. Installed Magento with Adobe Stock Integration
2. Enabled Adobe Stock Integration in _Stores - Configuration  - Advanced - System - Adobe Stock Integration_ but the API keys are not provided
3. Enhanced Media Gallery in Enabled 

### Steps to reproduce (*)
1. From Admin go to **Content** - **Pages**, click **Add New Page**
2. Expand **Content**,click **Show / Hide Editor**, click **Insert Image...** 
3. Click **Search Adobe Stock** button 
configuration_link_expected

1. Installed Magento with Adobe Stock Integration
2. Disabled Adobe Stock Integration in _Stores - Configuration  - Advanced - System - Adobe Stock Integration_ 
3. Enhanced Media Gallery in Enabled 

### Steps to reproduce (*)
1. From Admin go to **Content** - **Pages**, click **Add New Page**
2. Expand **Content**,click **Show / Hide Editor**, click **Insert Image...** 
3. Verify if the **Search Adobe Stock** button is available

### Expected result (*)
The Search Adobe Stock button is not available as the integration is disabled

### Steps to reproduce (*)
1. From Admin go to **Content** - **Pages**, click **Add New Page**
2. Expand **Content**,click **Show / Hide Editor**, click **Insert Image...**
3. Select a folder from the folder tree 
4. Click **Upload Image** button
5. Select an image file from your local machine, which has size over 2 MB, and click Open
6. Verify if the appeared _"cannot upload 'filename"... exceeds maximum file size limit."_ message disappears within several seconds

### Expected result (*)
The error message disappears

### CODE REVIEW

File: AdobeStockImageAdminUi/view/adminhtml/ui_component/media_gallery_listing.xml [hunk 1]
```diff
@@ -9,12 +9,7 @@
          xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Ui:etc/ui_configuration.xsd">
     <settings>
         <buttons>
-            <button name="search_adobe_stock">
-                <param name="on_click" xsi:type="string">jQuery(".adobe-search-images-modal").trigger("openModal");</param>
-                <param name="sort_order" xsi:type="number">100</param>
-                <class>media-gallery-actions-buttons</class>
-                <label translate="true">Search Adobe Stock</label>
-            </button>
+            <button class="Magento\AdobeStockImageAdminUi\Ui\Component\Listing\SearchAdobeStockButton" name="search_adobe_stock" />
```
Comment (coderimus) [inline AdobeStockImageAdminUi/view/adminhtml/ui_component/media_gallery_listing.xml:12]: I like the idea of the simplifying display button validation through the class of a button, but shouldn't we use modifiers for such case? @sivaschenko

**Review by coderimus — COMMENTED** [review]
@Nazar65 thank you for your contribution and improvements! I like it.

I have a question about the way the display of a search button validates. Let's wait for the @sivaschenko input.

**Comment by coderimus** [comment]
I set S1/P1 based on the #1326 priorities. @sivaschenko

**Comment by sivaschenko** [comment]
Only inventory MFTF have failed, they are not required currently

**Pull request merged**